platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/TlsFactory.java

```diff
@@ -123,4 +107,40 @@ public TlsFactory(
         clientSocket.startHandshake();
         return clientSocket;
     }
+
+    @Override
+    public void handlePeerListUpdate(final @NonNull List<PeerInfo> peers) {
```

Comment:
Not sure if this is in the coding standard, but within our team we usually put the nullity annotation before the `final` keyword.
```suggestion
    public void handlePeerListUpdate(@NonNull final List<PeerInfo> peers) {
```